src/cli/litra-toggle.ts

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env node
+
+import { findDevice, turnOff, turnOn } from "./../driver";
+
+try {
+  const device = findDevice();
+
+  if (!device) {
+    throw "Device not found";
+  }
+
+  const deviceArgKey = "-device";
+  const deviceArgVal = "camera";
+
+  const eventArgKey = "-event";
+  const eventArgVal = "on";
+
+  const deviceArg = process.argv[2] == deviceArgKey &&
+    process.argv[3] == deviceArgVal;
+  const eventArg = process.argv[4] == eventArgKey &&
+    process.argv[5] == eventArgVal;
+
+  if (deviceArg && eventArg) {
+    turnOn(device);
+  } else {
+    turnOff(device);
```

Comment:
I'm not sure that this script will do what you want it to do - but I could be wrong as I'm not a user of Oversight!

My understanding from the Oversight docs is that whenever the microphone turns on or off, Oversight will call this script with `deviceArg` to `microphone`. 

In that case, `deviceArg && eventArg` will evaluate to `false` and we'll go down this condition path, so your Litra will turn off. I don't think you want your Litra to turn off whenever the microphone state changes.